plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -501,6 +501,15 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 		})
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.SELinuxChangePolicy) {
+		addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "selinux-warning-controller"},
+			Rules: []rbacv1.PolicyRule{
+				eventsRule(),
```

Comment:
The new controller uses some informers to get pv, pvc, etc. We need to add some permissions for these resources.

Reply:
I thought that the controller uses shared informers and thus someone else needs those permissions, not the controller itself. But I can see other controllers include them. Added.